src/css/survey.css

```diff
@@ -40,7 +40,7 @@
 }
 
 p {
-  margin: 0 0 25px !important;
+  margin: 0 2px 25px !important;
```

Comment:
Instead of affecting all `p` tags, add one scoped to: `sv_q_radiogroup_control_label` since this class appears on the parent span of all radiogroup `p` tags

![image](https://user-images.githubusercontent.com/13375042/141493253-9f0f25e9-3074-4ab2-aef1-f6b1b7df161e.png)